builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -49,6 +49,26 @@ func TestAccAWSDBInstance(t *testing.T) {
 	})
 }
 
+func TestAccAWSDBInstanceReplica(t *testing.T) {
+	var s, r rds.DBInstance
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSDBInstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccReplicaInstanceConfig(rand.New(rand.NewSource(time.Now().UnixNano())).Int()),
```

Comment:
Can't we use the unique ID generator that Paul made recently https://github.com/hashicorp/terraform/pull/1632 ?